pylib/gyp/generator/cmake.py

```diff
@@ -616,9 +607,7 @@ def CreateCMakeTargetName(self, qualified_target):
     return base_name
 
 
-def WriteTarget(namer, qualified_target, target_dicts, build_dir, config_to_use,
-                options, generator_flags, all_qualified_targets, flavor,
-                output):
+def WriteTarget(namer, qualified_target, target_dicts, build_dir, config_to_use, options, generator_flags, all_qualified_targets, flavor, output):
```

Comment:
My personal rule is that if you have to left-right scroll in the GitHub window to read the whole line then the line should be wrapped.  Last I looked, that we 127 characters.

Reply:
IMHO those are code smells and I want them to be an eyesore, better then putting a `TODO`.